### ISSUE TITLE
Support for comma precision in number typecast

### ISSUE THREAD

**Issue by jarlah** [scotthovestadt/schema-object#47]
I have tested with this simple code, that the library does not typecast/transform a number with comma instead of dot in a number string.

```js
var SchemaObject = require('schema-object');

export const Person = new SchemaObject({
  firstName: String,
  lastName: String,
  age: Number,
  height: Number
});

const p =  new Person()
p.age = "32.345" // Works
p.age = "32,345" // Works NOT
```

Is it possible to support commas somehow in numbers? because, if it was supported this library would rock.

**Comment by jarlah** [comment]
why cant we supply an optional typecast function in the properties? For ex we have numberTransfrom ... we could also have typecast function which could be a function that returns a valid number

**Comment by scotthovestadt** [comment]
@jarlah Sorry, I didn't see this until now. It's probably related to the way it parses the number. I'll add support.

**Comment by scotthovestadt** [comment]
@jarlah Done. Added a test as well to guarantee future behavior. Releasing on NPM now.

**Comment by scotthovestadt** [comment]
Released as 4.0.2

**Comment by jarlah** [comment]
See PR #48 for what I actually meant :)
